samcli/lib/telemetry/metric.py

```diff
@@ -1,29 +1,35 @@
 """
 Provides methods to generate and send metrics
 """
+from dataclasses import dataclass
+
+from pathlib import Path
 from timeit import default_timer
 from functools import wraps, reduce
 
 import uuid
 import platform
 import logging
 import traceback
-from typing import Optional, Tuple
+from typing import Optional, Tuple, Dict
 
 import click
 
 from samcli import __version__ as samcli_version
 from samcli.cli.context import Context
 from samcli.cli.global_config import GlobalConfig
-from samcli.lib.warnings.sam_cli_warning import TemplateWarningsChecker
+from samcli.commands._utils.experimental import get_all_experimental_statues
```

Comment:
Nothing to do with this particular PR.

but note regardless: this file gets imported a LOT across all commands, so we need to be mindful of the imports we put in here and where possible rely on lazy-imports.